examples/word_of_day.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+const libingester = require('libingester');
+const mustache = require('mustache');
+const rp = require('request-promise');
+const template = require('./word_of_day_template');
```

Comment:
Eh I'm not really a fan of ithat approach since then you have to define your template in code rather than a separate file but it's not a big deal.